zone/client.h

```diff
@@ -181,6 +181,12 @@ struct XTarget_Struct
 	char Name[65];
 };
 
+typedef enum {
+	Killed_NPC,
+	Killed_DUEL,
+	Killed_PVP
+} KilledByTypes;
```

Comment:
Why are we using a `typedef enum` and not using type `KilledByTypes` instead of `uint8` in places?

Reply:
I think I addressed this with [`3ac5a41` (#3938)](https://github.com/EQEmu/Server/pull/3938/commits/3ac5a4188e12bd8ea60212bb69a5ee2e67d609e3)